docs/tutorials/projects/verification-system.md

```diff
@@ -17,11 +17,12 @@ no_list: true
 # SMEs: Bijan Haney
 ---
 
-In the [registry](https://app.viam.com/module/viam-labs/verification-system), Viam Labs provides a [vision service classifier](/ml/vision/classification/) model called `verification-system`.
-Follow this tutorial to learn how to configure this module and set up your own security system to detect, verify, and alarm based on specified detections from nested [vision service detectors](/ml/vision/detection/).
+Follow this tutorial to create a simple security system with visual people detection to alarm, and facial recognition to disarm.
+In this project, you will create a security system with facial recognition to "verify" whether intruders present are recognized by your system and can disable your alarm.
+To accomplish this on your machine, you will configure an [`mlmodel` person detector](#configure-an-mlmodel-person-detector) and [facial detector](#configure-a-facial-detector) together in the [`verification-system` module](https://app.viam.com/module/viam-labs/verification-system).
```

Comment:
- I think this goes beyond simple so let's just call it a security system
- please start this in even simpler terms - you're starting the first sentence with alarming and disarming but what does this mean. 
- you use the words "intruders present" but they're just people at that point, if they're not recognized, then they're intruders. This is confusing.

You can try walking the reader through what happens when the system is set up and a person walks into view. What does the machine check and then what does it do.

Reply:
Rephrased. I think at the last point that's what I'm trying to do on lines 25-26?